101-lab/content/00_overview.md

```diff
@@ -1,31 +1,42 @@
 # Overview
 Welcome!
 
-
 In these labs you will explore builds, deployments, as well as some common issues with storage provisioning.
 
 These labs will focus on a single 2-tier application based on the popular open source chat utility, Rocket.Chat.
 
 ## Special Notes
 
+The training is being delivered in a production Openshift Environment, using the BC Gov's 'Silver' cluster. 
 
-The training is being delivered in a production Openshift Environment.
-
+You'll need an IDIR account to complete this course, so if you don't have one please contact your supervisor to arrange one before starting the course.
 
 ## Course Format
 
-This course has been designed to be run _at your own pace_. The lab materials have been designed so that you generally follow the happy path for a simple two-tier deployment. There will be many intentional bugs along the way that will have you encounter real-life troubleshooting techniques!
+We offer this course in two modes, 'live training' or 'self-paced'. 
 
-The course length can vary (based on scheduled). It was designed to be run over a period of a few days rather than just a day or two to have time for isolated touchpoints as well as to provide time for discussion.  During that time __touch points__ will be scheduled with the cohort to talk about special topics and to assist in troubleshooting. 
+If you're attending the live training, we'll run through some background information in a 1-day workshop before the lab begins. 
 
-### This course is new!
+For the self-paced training, you can view the slides and recordings of them being presented [here](https://docs.google.com/presentation/d/1VNxqtRRECNd4uzZZslpLsdKYYXN70Gpu-jCLCD1yF9Y/edit?usp=sharinghere). 
```

Comment:
I can edit the content from this link,  not sure if it's the same for anyone outside of our team? If so, let's switch it to a view only acces

Reply:
That's a special power for you and the rest of our training facilitators, Iryna just set it up this morning. I've just double checked from incognito and have only view access.